crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -41,12 +41,18 @@ impl<I, V> SparseArray<I, V> {
 macro_rules! impl_sparse_array {
     ($ty:ident) => {
         impl<I: SparseSetIndex, V> $ty<I, V> {
+            /// Checks if a value is present at `index`.
```

Comment:
Genuinely asking, are these useful comments? Wouldn't people know what the contains function does just from a glance at the type signature?

Reply:
Perhaps, but this is definitely needed for us to enable `cfg!(warn(missing_docs))` without breaking CI.